README.md

```diff
@@ -77,7 +77,7 @@ map.whenReady(function() {
 * [rotation](#rotation)
 * [selected](#selected)
 * [suppressToolbar](#Suppress-Toolbar)
-
+*  [tooltipText](#Tooltip-Text)
```

Comment:
Hello @sudeep162002 Could you remove one space before the text
like so `* [tooltipText](#Tooltip-Text)` not `*  [tooltipText](#Tooltip-Text)`
Thank you!

Reply:
done

Reply:
Great work @sudeep162002